docs/threat-modeling-for-designers/threat-modeling-for-designers.md

```diff
@@ -0,0 +1,54 @@
+# Threat Modeling for Designers
+
+This is a living, work-in-progress documentation to shape how might _Threat Modeling_ (borrowing from security field) look like for designers. Alternatively, we can relabel "threat modeling" as "risk assessment."
```

Comment:
'threat modeling' as a concept might not be comfortable for a designer. 💡 Ben P.

Reframing: how do we avoid doing the bad thing, framing of design strategies fo building safer platforms.

Tying together between visual/strategist (not someone with security background) and threat modeling (on a surface).

Reply:
Added a note to address this and revised.